apps/src/sharedComponents/SystemDialog/ConfirmCancelFooter.jsx

```diff
@@ -15,21 +15,32 @@ export default class ConfirmCancelFooter extends React.Component {
   static propTypes = {
     onConfirm: PropTypes.func.isRequired,
     onCancel: PropTypes.func.isRequired,
-    confirmText: Button.propTypes.text,
-    cancelText: Button.propTypes.text,
-    confirmColor: Button.propTypes.color.isRequired,
-    cancelColor: Button.propTypes.color.isRequired,
+    confirmText: PropTypes.string,
+    cancelText: PropTypes.string,
+    confirmColor: PropTypes.oneOf([
+      'purple',
```

Comment:
It looks like we have a constant called buttonColors in the Button file- would it be possible to use that here?